.github/workflows/auto-rebuild.yml

```diff
@@ -0,0 +1,38 @@
+name: GDevelop PR automatic rebuild of the extensions database
+on:
+  pull_request:
+    types: [opened,synchronize]
+jobs:
+  rebuild-database:
+    # Only run it on extensions submissions (tag is ensured via the new issue form)
+    if: ${{ contains(github.event.pull_request.labels.*.name, '✨ New extension') || contains(github.event.pull_request.labels.*.name, '🔄 Extension update') }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Rebuild the database
+        run: |
+          npm i
+          echo 'BUILD_LOGS<<EOF' >> $GITHUB_ENV
+          node scripts/generate-extensions-registry.js --disable-exit-code >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+          echo "CHANGED="$(git diff --name-only extensions-registry.json) >> $GITHUB_ENV
+
+      - name: Commit and push if changed
+        if: ${{ contains(env.CHANGED, 'extensions-registry') }}
+        run: |
+          git config --global user.name '${{ github.actor }}'
+          git config --global user.email '${{ github.actor }}@users.noreply.github.com'
+          git add extensions-registry.json
+          git commit -m "Regenerated extension registery"
+          git push
```

Comment:
This step never triggers for a reason I cannot find. I've left it here as it cannot hurt and may make it easier to add this in if someone can figure out why it does not trigger.

Reply:
Ok, we'll probably remove this json soon as only useful for old GDevelop versions.

Reply:
@4ian not sure what you mean, the `extensions-registry.json` is still being used by the latest version of GDevelop. (fetched and enriched)

We could modify our Extension service to fetch the extensions from the `/Extensions` folder rather than this json, and generate it for the app. Is this what you have in mind?

Reply:
The latest versions are using https://resources.gdevelop-app.com/extensions-database/extensions-database.json - which is generated inside the `dist` folder when you run `npm run build` locally (and then deployed).
But not extensions-registry.json directly :) (unless I missed something else). This file is still in the git because some versions ago the registry was directly fetched by the editor from github. But now all calls should be going through the API, and from the very latest version it is even returned directly by the API (rather than a redirection).

Reply:
ooooh right :+1: I misread the name of the file :)